### ISSUE TITLE
Module idea:Filters for image-sequencer

### ISSUE THREAD

**Issue by Harshithpabbati** [publiclab/image-sequencer#828]
### Lets add few filters for the image sequencer.

**Comment by HarshKhandeparkar** [comment]
Cool! This can be another meta-module issue. The metaModule can be created from the convolution module

**Comment by 28-ys** [comment]
hi, can i work on this ?

**Comment by 28-ys** [comment]
hi @HarshKhandeparkar  @harshithpabbati  can i work on this ?

**Comment by HarshKhandeparkar** [comment]
Sure!

**Comment by ankushshukla** [comment]
Hi @HarshKhandeparkar  @harshithpabbati  sorry I've not text and discuss in last weeks cause of some health issues, well basically our motive to add new module right ?

**Comment by 28-ys** [comment]
Hi there, 
@harshithpabbati  @HarshKhandeparkar  sorry for not discuss in last few days cause of some health issue, well basically our motive is to add new module here right ?

**Comment by 28-ys** [comment]
Hi 
@HarshKhandeparkar @harshithpabbati  well basically our motive is to add new module here right ?

**Comment by HarshKhandeparkar** [comment]
Yes. New module or modules.

**Comment by 28-ys** [comment]
So, module should be crisp, dulled, toy and flash of type or i can work and discuss my ideas as well for new module.

**Comment by HarshKhandeparkar** [comment]
Everyone's ideas are welcome. This is open source!

**Comment by 28-ys** [comment]
Hi there,
@HarshKhandeparkar  I've find two ways to sketch module...
to using [' canvas-sketch-cli '] npm module , 
or I've find another way is to add Greyscale--> it will make the image black and white, then we'll apply invert and blur function on it which we already have just need to import it, then finally Dodge and Merge --> basically it is  blend mode divides the bottom layer by the inverted top layer. then lightens the bottom layer depending on the value of the top layer which highlights the boldest edges. 
and check this final result in image... and guide me. 
Thanks.
[Screenshot 2020-02-03 at 1.22.02 PM.pdf](https://github.com/publiclab/image-sequencer/files/4146768/Screenshot.2020-02-03.at.1.22.02.PM.pdf)

**Comment by blurry-x-face** [comment]
@28-ys there is already a open PR for sketch module

**Comment by 28-ys** [comment]
Hi @HarshKhandeparkar  @harshithpabbati

**Comment by 28-ys** [comment]
@blurry-x-face  so do you think my idea will work ?

**Comment by blurry-x-face** [comment]
Yes, I was trying to make sketch module using the same method and was able to make it succesfully.

**Comment by 28-ys** [comment]
@blurry-x-face  Congrats! which way you apply by using ['canvas-sketch-cli'] or by adding Greyscale.
Please give me some guide for either add new feature or anything.  
I actually new to open source and want to contribute. 
Thanks!

**Comment by blurry-x-face** [comment]
I did it by following these steps:
```
1. Greyscale
2. Invert
3. Blur
4. Blending with color-dodge
```

>I actually new to open source and want to contribute.

Have you solved any fto?

**Comment by 28-ys** [comment]
@blurry-x-face  no i haven't solved just find a way to add new module for sketch.

**Comment by 28-ys** [comment]
@blurry-x-face 
No i haven't solve any fto.

**Comment by blurry-x-face** [comment]
try solving #1527

**Comment by 28-ys** [comment]
@blurry-x-face 
I guess #1527 is already done cause it is there below NEED HELP

**Comment by 28-ys** [comment]
Hi,
@blurry-x-face 
I guess #1527 is already done cause it is there below NEED HELP.
